src/datatrove/tools/jobs_status.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import json
+import os.path
+import re
+
+from loguru import logger
+from rich.console import Console
+
+from datatrove.io import get_datafolder
+from datatrove.utils._import_utils import is_rich_available
+
+
+def list_folders_with_prefix(log_files_path, log_prefix):
+    # Get a list of all folders in the given path
+    folders = [folder for folder in os.listdir(log_files_path) if os.path.isdir(os.path.join(log_files_path, folder))]
+    # Filter out only the folders that start with the specified prefix
+    folders_with_prefix = [os.path.join(log_files_path, folder) for folder in folders if folder.startswith(log_prefix)]
+
+    return folders_with_prefix
+
+
+if not is_rich_available():
+    raise ImportError("Please install `rich` to run this command (`pip install rich`).")
+
+
+parser = argparse.ArgumentParser("Fetch all jobs that are running or complete.")
+
+parser.add_argument(
+    "path", type=str, nargs="?", help="Path to the logging folder. Defaults to current directory.", default=os.getcwd()
+)
+
+parser.add_argument("--log_prefix", type=str, nargs="?", help="Prefix of logging folders to be scanned.", default="")
+parser.add_argument("--show_complete", help="Also list all jobs that are already complete.", action="store_true")
```

Comment:
wouldn't it make sense to show them by default?

Reply:
No strong opinion, in my case the complete ones were a lot so they polluted the list :)

Reply:
changed it!

Reply:
I think it's a bit confusing now, maybe just `hide_complete` with `store_false`?